Formula/README.md

```diff
@@ -0,0 +1,39 @@
+# Create `brew` formula
```

Comment:
The same question as it was for snap: is it possible to name this folder `/Formula` -> `/developer/brew`? Or this `Formula` is required by Brew's conventions?

Reply:
It seems like it's required that the formula (`*.rb`) needs to reside inside of a `/Formula` directory. Another option would be to use a separate GithHub repo `strictdoc-project/homebrew-strictdoc` instead. Again it seems like the folder structure needs to be satisfied. An example repo is [`simonw/hoomebrew-datasette`](https://github.com/simonw/homebrew-datasette).

Reply:
Thanks for explaining. I think `Formular` is ok then, no need to create a separate repo.